portfolio/src/main/webapp/index.html

```diff
@@ -52,6 +57,11 @@ <h2 class="wrapper">
     <!-- Note: each list item is a floating box in the background -->
     <div class="animation-area">
         <div class="title">Hi, I'm Jamari.</div>
+        <div  class="randomMessageDiv">
+            <p id="random-message-txt" class="randomMessageTxt">Click the button to get a fact about me!</p> 
```

Comment:
nit: formatting :)

Reply:
oh yeah I forgot that cloud shell handles formatting really nicely for us with just one button :o